src/utils/evm/accountHandler.ts

```diff
@@ -245,6 +246,10 @@ class EVMAccountHandler {
     return [rpcStore.useGasless ? scwInstance.scwAddress : this.wallet.address]
   }
 
+  getPrivateKey(): string {
+    return '0x' + userStore.privateKey
+  }
```

Comment:
???

Shouldn't we check if `userStore.privateKey?.startsWith('0x')` if this is false only then we need to append '0x' right?

Reply:
why question marks?